### DISCUSSION TITLE
Export table as csv feature is not working version v0.10.5

### DISCUSSION THREAD

**Discussion by multispeed** [amidaware/tacticalrmm#891] [Q&A] [answered]
Hi,

I would like to export a CSV from all the software installed on a particular system, but the blue export button isn't providing me with a csv. i Then tried the button on another page (debug) and the button isn't working there either.

I could find any other issue in Discussions or a clue why it isn't working in the Documentation.

**Comment by wh1te909** [comment, marked as answer]
thanks this is a bug, issue has been opened https://github.com/wh1te909/tacticalrmm/issues/895
